packages/nc-gui/components/cell/MultiSelect.vue

```diff
@@ -52,6 +53,11 @@ const isPublic = inject(IsPublicInj, ref(false))
 
 const isForm = inject(IsFormInj, ref(false))
 
+const rowHeight = inject(
+  RowHeightInj,
+  computed(() => undefined),
```

Comment:
You can use default value as `ref(undefined)` and change type definition to `RowHeightInj: InjectionKey<Ref<1 | 2 | 4 | 6 | undefined>>`(I guess ComputedRef is a subclass of Ref)